Pipfile

```
@@ -8,12 +8,14 @@ name = "pypi"
 [dev-packages]
 autopep8 = "*"
 black = "*"
+cookiecutter = "*"
 flake8 = "*"
 flake8-docstrings = "*"
 isort = "*"
 mypy = "*"
 pre-commit = "*"
 pytest = "*"
+taipy = "*"
```

Since the unit test runs the application created by the template, installing taipy is required.

I'm not sure if we should specify the version here. My guess is whether the latest version, or the dev version.

---

If use develop version of taipy:

```suggestion
taipy = {git = "https://git@github.com/avaiga/taipy.git@develop"}
```